src/main/java/com/sinthoras/visualprospecting/mixins/ifu/ItemOreFinderToolMixin.java

```diff
@@ -0,0 +1,64 @@
+package com.sinthoras.visualprospecting.mixins.ifu;
+
+import com.encraft.dz.items.*;
+import com.sinthoras.visualprospecting.*;
+import com.sinthoras.visualprospecting.database.*;
+import gregtech.api.enums.*;
+import gregtech.api.objects.*;
+import java.util.*;
+import java.util.stream.*;
+import net.minecraft.block.*;
+import net.minecraft.entity.*;
+import net.minecraft.entity.player.*;
+import net.minecraft.item.*;
+import net.minecraft.world.*;
+import org.spongepowered.asm.mixin.*;
+import org.spongepowered.asm.mixin.injection.*;
+import org.spongepowered.asm.mixin.injection.At.*;
+import org.spongepowered.asm.mixin.injection.callback.*;
+
+@Mixin(value = ItemOreFinderTool.class, remap = false)
+public class ItemOreFinderToolMixin extends Item {
+
+  @Shadow
+  private static int found;
+  
+  @Shadow
+  private static int MAX_DAMAGE;
+
+  @Inject(
+      method = "onUpdate",
+      at = @At(value = "INVOKE", target = "Lcom/encraft/dz/items/ItemOreFinderTool;shouldKeepLooking()Z", shift = Shift.AFTER, remap = false),
+      locals = LocalCapture.CAPTURE_FAILSOFT,
```

Comment:
FAILHARD

Reply:
So it won't be loaded if mod absent, right?

Reply:
No. That is taken care of by the MixinPlugin. It is only loaded when the mod has been detected (via jar name)